assets/css/main.css

```diff
@@ -1,9 +1,8276 @@
-﻿/*!
```

Comment:
Is this intentional that `main.css` seems to contain the whole Bootstrap CSS? This is not the built minimized output of SASS.

Reply:
That is weird, I thought I ran `drone exec`. Apparently committed at the wrong phase then. In any case, thanks for pointing out, this has now been fixed!

Reply:
`main.css` always contains the bootstrap stuff, but that should be in a single line normally. It sometimes could happen that one runs drone exec, but has the sass watch container running at the same time, and then they touch a source file, which triggers a rebuild, which overwrites the `main.css` again. One thing is to run `git diff --cached` to verify that you are committing what you think you are. However, `main.css` doesn't format there so well.

Reply:
Good to know!